.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Test
+
+on:
+  push:
+  pull_request:
+    types: [ opened, synchronize, edited, reopened ]
+
+jobs:
+  test:
+    name: Linter and Unit test
+    runs-on: macos-latest
+    steps:
+      - name: Git Checkout
```

Comment:
you may want to add this action the cancel previous runs 😁 
```
    - name: Cancel Previous Runs
       uses: styfle/cancel-workflow-action@0.5.0
       with:
         access_token: ${{ github.token }}
```

Reply:
Thank you. I'll make that change 😊